downloads/swan-lake-release-notes/swan-lake-2201.8.3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake Update 8 (2201.8.3) 
+permalink: /downloads/swan-lake-release-notes/2201.8.3/
+active: 2201.8.3
+redirect_from: 
```

Comment:
```suggestion
redirect_from: 
```
Need to remove these redirections as they are not used anymore. Please check the template.